composer.json

```diff
@@ -26,35 +26,36 @@
         "doctrine/common": "^2.7",
         "doctrine/inflector": "^1.4 || ^2.0",
         "doctrine/persistence": "^1.3.3",
-        "knplabs/knp-menu-bundle": "^2.2.2",
+        "knplabs/knp-menu": "^3.1",
+        "knplabs/knp-menu-bundle": "^3.0",
         "sonata-project/block-bundle": "^3.20 || ^4.0",
         "sonata-project/exporter": "^1.11 || ^2.0",
         "sonata-project/form-extensions": "^0.1.1 || ^1.4",
         "sonata-project/twig-extensions": "^0.1.1 || ^1.3",
-        "symfony/asset": "^4.4",
-        "symfony/config": "^4.4",
-        "symfony/console": "^4.4",
-        "symfony/dependency-injection": "^4.4.3",
-        "symfony/doctrine-bridge": "^4.4",
-        "symfony/event-dispatcher": "^4.4",
+        "symfony/asset": "^4.4 || ^5.1",
```

Comment:
What's the reason for `^4.4 || ^5.1` on Symfony packages rather than `^4.4 || ^5.0`? Is there something hard to support in 5.0?

Reply:
Symfony 5.0 is EOL (not supported anymore). See https://symfony.com/releases

So does not really make sense to support it here I believe :blush:

Reply:
So it is, sorry 🤦